### PR TITLE
Fix #126: Added debugging information for Lighthouse Tests

### LINKED ISSUE #126: Develop and document debugging tools and procedures for lighthouse
debugging instructions is not present in the wiki for lighthouse test failures

Link: [https://github.com/oppia/oppia/wiki/Lighthouse-Tests#debugging-lighthouse-tests](https://github.com/oppia/oppia/wiki/Lighthouse-Tests#debugging-lighthouse-tests)

### PR DESCRIPTION
**Pull request by Patel-Muhammad**
This PR adds the procedure to debug the failing lighthouse tests.

### CODE REVIEW

**Comment by Patel-Muhammad** [comment]
Hi @Rijuta-s can you please take a look at this PR and let me know if it requires any changes. Thanks!

File: Lighthouse-Tests.md [hunk 1]
```diff
@@ -78,3 +78,36 @@ def export_url(url):
...
+
+## Steps solve the failing lighthouse test
+
+**1. Find out which pages are failing:** You can find which pages are failing in the lighthouse logs
+<img height="150" src="https://github.com/Patel-Muhammad/oppia-web-developer-docs/assets/96219910/a7833e61-8294-446e-a588-bdfbef13ab10">
+
+**2. Find report for failing page:** The lighthouse checks will upload reports to a temporary url for each page that it audits. 
+The lighthouse checks will print the link to each report in the logs.
+
+![image](https://github.com/Patel-Muhammad/oppia-web-developer-docs/assets/96219910/8e5a712c-5409-4e5c-90b6-7c2bb9ad21c3)
```
Comment (Ash-2k3) [inline Lighthouse-Tests.md]: Hi @Patel-Muhammad, all the images seem to be in your fork. Can you add them to the oppia/oppia-web-developer-docs repo instead?

**Review by Ash-2k3 — CHANGES_REQUESTED** [review]

**Comment by Rijuta-s** [comment]
@Patel-Muhammad please assign @Lawful2002 and get in touch with him for this PR

File: Lighthouse-Tests.md (hunk 1 above)
Comment (Patel-Muhammad) [reply Lighthouse-Tests.md]: Hi @Ash-2k3 can you please let me know how can I do that? Should I add the images to the images folder that is present?

**Comment by Patel-Muhammad** [comment]
Hi @Lawful2002 Can please let me know how can  add the images to the oppia/oppia-web-developer-docs repo? Thanks

File: Lighthouse-Tests.md (hunk 1 above)
Comment (Ash-2k3) [reply Lighthouse-Tests.md]: Yes that is right, also this is not the first time we are adding images in the wiki. You may take a look at wikis which already has images and mimic the method of adding images.

Comment (Patel-Muhammad) [reply Lighthouse-Tests.md]: Hi @Ash-2k3 I have done that PTAL!

**Comment by Lawful2002** [comment]
@Patel-Muhammad have you been able to figure this out? (Sorry for the late response)

**Comment by Patel-Muhammad** [comment]
Hi @Lawful2002 I have down the changes PTAL. Thanks!

**Comment by Lawful2002** [comment]
@Ash-2k3 PTAL, thanks!

**Comment by Ash-2k3** [comment]
Thanks @Patel-Muhammad, LGTM

**Review by Ash-2k3 — APPROVED** [review]

**Pull request merged**